tsdat/const/__init__.py

```diff
@@ -1,6 +1 @@
-from .const import COORDINATE_SYSTEM
-from .const import INPUT_DATASTREAM
-from .const import OUTPUT_DATASTREAM
-
-from .const import InputKey
-from .const import VarName
+from .const import *
```

Comment:
Star imports are discouraged by the linter I use (ruff). The recommended approach is to do this:

```python
from .const import COORDINATE_SYSTEM as COORDINATE_SYSTEM
from .const import INPUT_DATASTREAM as INPUT_DATASTREAM
from .const import OUTPUT_DATASTREAM as OUTPUT_DATASTREAM
from .const import InputKey as InputKey
from .const import VarName as VarName
```